common/src/main/java/io/netty/util/internal/ThrowableUtil.java

```diff
@@ -76,4 +78,12 @@ public static void addSuppressed(Throwable target, List<Throwable> suppressed) {
             addSuppressed(target, t);
         }
     }
+
+    @SuppressJava6Requirement(reason = "Throwable getSuppressed is only available for >= 7. Has check for < 7.")
+    public static Throwable[] getSuppressed(Throwable source) {
+        if (!haveSuppressed()) {
```

Comment:
Not necessarily part of this PR but this method should be named `hasSuppressed()`.

Reply:
we can do the as another PR

Reply:
Sure.